docs/unified/clevo/post-install.md

```diff
@@ -75,6 +75,50 @@ Select your operating system to view applicable instructions:
     curl -sSf https://raw.githubusercontent.com/Dasharo/dasharo-tools/main/clevo/sata-suspend-fixup | sudo sh
     ```
 
+=== "Qubes OS"
+    ### Touchpad hotkey enablement
+
+    The touchpad hotkey may need extra setup to function correctly under Qubes.
+    If the key isn't working, follow the steps below:
+
+    1. Open dom0 terminal window
+
+    1. Open /etc/udev/hwdb.d/60-keyboard.hwdb in your preferred text editor,
+       e.g. nano:
+
+        ```bash
+        sudo nano /etc/udev/hwdb.d/60-keyboard.hwdb
+        ```
+
+    1. Type the following into the file:
+
+        !!! note
+            The following applies only to NV4x 12th Gen and V560TU, which are
+            currently the only officially Qubes-certified models. If you use
+            Qubes on any other model, you may need to adjust the SMBIOS product
+            name in the section below.
+
+        ```
+        evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNV4xPZ:*
+        evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnnovacustom-v560tu:*
```

Comment:
We need to fix the product name in our heads build to match UEFI, right now it doesnt

Reply:
Fixed in https://github.com/Dasharo/docs/pull/974/commits/9e03bcfebf280e6b7a8592c35c6eac9c97da6d92

matching FW change: https://github.com/Dasharo/heads/commit/a93c1766810e4733f8538d4910a85c44e0c7d8c9#diff-1aa063ec528a5753e685e3e8c0d35336015074f1746351e1a221b177e999f622R20